src/Resources/npm-package/js/contao-filter-bundle.js

```diff
@@ -72,16 +72,18 @@ class FilterBundle {
     /**
      * Register events for form async submit.
      *
-     * These events need to be reintialized after form reload.
+     * These events need to be reinitialized after form reload.
      */
     static registerFormAsyncSubmitEvents()
     {
-        document.querySelector('.mod_filter form[data-async] input[type="text"]').addEventListener('keydown', (event) => {
-            if (event.key === 'Enter') {
-                event.preventDefault();
-                event.stopPropagation();
-                FilterBundle.asyncSubmit(event.target.form);
-            }
+        document.querySelectorAll('.mod_filter form[data-async] input[type="text"]').forEach(function(element) {
+            element.addEventListener('keydown', (event) => {
```

Comment:
You are declaring these funtions in 2 ways, just for codestyle purpose i would prefer to use the (element) => {} notation in both cases

Reply:
@AlexejKossmann Good suggestion, is fixed!